### PR TITLE
Update and test exporters

### PR DESCRIPTION
**Pull request by caspervdw**
This repairs and tests the PyAV exporter, as well as the moviepy movie exporter. Many tests could be designed still, but it's a start.

### CODE REVIEW

**Comment by caspervdw** [comment]
This is done, if test pass can be merged.

**Comment by caspervdw** [comment]
@danielballan  Could you check up on this and merge if you agree? Then I will make the release notes and we can release v0.4

File: pims/display.py [hunk 1]
```diff
@@ -33,8 +34,9 @@
 
 
 def export_pyav(sequence, filename, rate=30, bitrate=None,
-                width=None, height=None, codec='mpeg4', format='yuv420p',
-                autoscale=True):
+                width=None, height=None, format=None, codec='mpeg4',
+                pixel_format='yuv420p', autoscale=None, quality=None,
```
Comment (danielballan) [inline pims/display.py]: Would you note the parameter name change `format` -> `pixel_format` in the release notes?

File: pims/display.py [hunk 2]
```diff
@@ -160,25 +202,29 @@ def play(sequence, rate=30, bitrate=None,
...
+            return self._cache[:, :, ::-1]
+        else:
+            return self._cache
 
 
 def export_moviepy(sequence, filename, rate=30, bitrate=None, width=None,
-                   height=None, codec='libx264', format='yuv420p',
-                   autoscale=True, quality=None, verbose=True,
-                   ffmpeg_params=None, rate_range=(16, 32)):
+                   height=None, codec='mpeg4', pixel_format='yuv420p',
```
Comment (danielballan) [inline pims/display.py]: Ditto here.

**Comment by danielballan** [comment]
I now see that you mentioned release notes above. I'll merge this as is and let you address the release notes separately.

**Comment by danielballan** [comment]
...And I see that `format` -> `pixel_format` was documented in #261, just not in this branch (which is where I looked).

**Pull request merged**